.travis.yml

```diff
@@ -55,7 +57,9 @@ before_deploy: |
 deploy:
   provider: releases
   api_key: "$GITHUB_RELEASE_TOKEN"
-  file: "$SLANG_BINARY_ARCHIVE"
+  file: 
+    - "$SLANG_BINARY_ARCHIVE"
+    - "$SLANG_BINARY_ARCHIVE_TAR"
```

Comment:
Not sure if this is the right way to deploy multiple files.

Reply:
Looks correct according to the [Travis documentation](https://docs.travis-ci.com/user/deployment/releases/#uploading-multiple-files).